Source/Scene/GroundPrimitive.js

```diff
@@ -0,0 +1,655 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/GeometryInstance',
+        '../Core/isArray',
+        '../Core/Math',
+        '../Core/Matrix4',
+        '../Renderer/DrawCommand',
+        '../Renderer/ShaderSource',
+        '../Shaders/ShadowVolumeFS',
+        '../Shaders/ShadowVolumeVS',
+        '../ThirdParty/when',
+        './BlendingState',
+        './DepthFunction',
+        './Pass',
+        './PerInstanceColorAppearance',
+        './Primitive',
```

Comment:
Do you think we should move `_createColumbusViewShader` and friends to a new private class instead of having this include `Primitive`?